src/proj.rs

```diff
@@ -1186,7 +1186,7 @@ mod test {
         assert_eq!(area.west, -35.58);
         assert_eq!(area.south, 24.6);
         assert_eq!(area.east, 44.83);
-        assert_eq!(area.north, 84.17);
```

Comment:
This is a pretty large change - curious!

Reply:
It certainly is, but this isn't even a value we're calculating. Perhaps I'll ask on the list…

Reply:
Extent changed January 2021: https://epsg.org/closed-change-request/browse/id/2020.127
New extent, matching what we now get from the DB: https://epsg.org/extent_2881/Europe-LCC-LAEA.html
CRS: https://epsg.org/crs_3035/ETRS89-extended-LAEA-Europe.html

Reply:
The fast moving world of geodesy!

Fascinating.